src/libraries/System.Runtime.Caching/Directory.Build.props

```diff
@@ -6,6 +6,8 @@
          to a different assembly. -->
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IncludePlatformAttributes>true</IncludePlatformAttributes>
+    <UnsupportedOSPlatforms>browser</UnsupportedOSPlatforms>
```

Comment:
this is not correct, it works on browser with my PR from yesterday.

Reply:
Thanks for spotting. Fixed.